AISKU/Tests/applicationinsights.e2e.tests.ts

```diff
@@ -61,6 +61,7 @@ export class ApplicationInsightsTests extends TestClass {
                 disableFetchTracking: false,
                 enableRequestHeaderTracking: true,
                 enableResponseHeaderTracking: true,
+                includeResponseErrorData: true,
```

Comment:
I think this should be renamed to `disableAjaxErrorStatusText` since it seems like something we would want to enable by default.